pyanaconda/payload/dnf/payload.py

```diff
@@ -275,125 +275,6 @@ def _replace_vars(self, url):
 
         return None
 
-    def _add_repo(self, ksrepo):
```

Comment:
Is this refactorization necessary?

Reply:
It's not necessary but also it's not really dangerous. It's just easier to find the pieces in the code. However, if you insist on this I can definitely drop the first commit.

Reply:
Fixed.